openfisca_france/parameters/impot_revenu/cehr.yaml

```
@@ -1,23 +1,48 @@
-brackets:
-- rate:
-    2011-01-01:
-      value: 0.0
-  threshold:
-    2011-01-01:
-      value: 0.0
-- rate:
-    2011-01-01:
-      value: 0.03
-  threshold:
-    2011-01-01:
-      value: 250000.0
-- rate:
-    2011-01-01:
-      value: 0.04
-  threshold:
-    2011-01-01:
-      value: 500000.0
 description: Contribution exceptionnelle sur les hauts revenus
-metadata:
-  threshold_unit: currency
-  rate_unit: /1
+reference: "http://www.legifrance.gouv.fr/affichCode.do?cidTexte=LEGITEXT000006069577&idSectionTA=LEGISCTA000025049019"
+celibataire_ou_assimile:
+  metadata:
+    threshold_unit: currency
+    rate_unit: /1
+  brackets:
+  - rate:
+      2011-01-01:
+        value: 0.0
+    threshold:
+      2011-01-01:
+        value: 0
+  - rate:
+      2011-01-01:
+        value: 0.03
+    threshold:
+      2011-01-01:
+        value: 250_000
+  - rate:
+      2011-01-01:
+        value: 0.04
+    threshold:
+      2011-01-01:
+        value: 500_000
+maries_ou_pacses:
+  metadata:
+    threshold_unit: currency
+    rate_unit: /1
+  brackets:
+  - rate:
+      2011-01-01:
+        value: 0.0
+    threshold:
+      2011-01-01:
+        value: 0
+  - rate:
+      2011-01-01:
+        value: 0.03
+    threshold:
+      2011-01-01:
+        value: 500_000
+  - rate:
+      2011-01-01:
+        value: 0.04
+    threshold:
+      2011-01-01:
+        value: 1_000_000
```

Résoudre le ⛔ en ajoutant un retour à la ligne ?
```suggestion
        value: 1_000_000

```